build.gradle.kts

```diff
@@ -32,9 +32,13 @@ dependencies {
 	implementation("org.slf4j:slf4j-api:2.0.9")
 	implementation("org.springframework.ai:spring-ai-openai-spring-boot-starter")
 	implementation(platform("org.springframework.ai:spring-ai-bom:1.0.0-SNAPSHOT"))
+	implementation ("io.temporal:temporal-sdk:1.20.1")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit5")
     testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+	testImplementation ("io.temporal:temporal-testing:1.17.0")
+	implementation("io.temporal:temporal-spring-boot-starter:1.25.1")
```

Comment:
I dont understand why i have 8 white spaces 
In project everyting looks good

![image](https://github.com/user-attachments/assets/ea6584ba-f22e-4a5d-b965-efedf7b3d2bf)

Reply:
pre-commit hook perhaps?